jito-protos/Cargo.toml

```diff
@@ -10,7 +10,7 @@ prost = "0.11.9"
 prost-types = "0.11.9"
 solana-perf = "=1.14.18"
 solana-sdk = "=1.14.18"
-tonic = "0.9.2"
+tonic = "0.8.3"
```

Comment:
curious does tonic 0.9 have conflicts?

Reply:
yeah, the geyser import fucks things up :/